<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comptes</class>
 <widget class="QDialog" name="comptes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="id_comptes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenom">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mail">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="grade">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>grade</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>441</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modifier">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>362</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Effectuer">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 18pt &quot;DEAR Daddy PERSONAL USE&quot;;
background:rgba(255,255,255,0);
border-radius:8px;
}
QPushButton::hover{
background:rgba(0,0,0,85);
}</string>
   </property>
   <property name="text">
    <string>Effectuer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rechercher</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_recherche">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tri">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>42</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1254</width>
     <height>600</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="interface.qrc">:/new/prefix1/interface/79303081_559980864545992_7566787920833544192_n (1).jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="retour">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:transparent; </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="statistique">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistique</string>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>id_comptes</zorder>
  <zorder>nom</zorder>
  <zorder>prenom</zorder>
  <zorder>mail</zorder>
  <zorder>password</zorder>
  <zorder>grade</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>tableView</zorder>
  <zorder>ajouter</zorder>
  <zorder>pushButton_modifier</zorder>
  <zorder>Effectuer</zorder>
  <zorder>supprimer</zorder>
  <zorder>label_7</zorder>
  <zorder>lineEdit_recherche</zorder>
  <zorder>pushButton_tri</zorder>
  <zorder>retour</zorder>
  <zorder>statistique</zorder>
 </widget>
 <resources>
  <include location="interface.qrc"/>
 </resources>
 <connections/>
</ui>
